src/main/native/ECKey.c

```diff
@@ -1129,7 +1129,7 @@ JNIEXPORT jlong JNICALL Java_com_ibm_crypto_plus_provider_ock_NativeInterface_XE
     if ( debug ) {
       gslogMessage ("DATA_XEC size=%d", (int) size);
       gslogMessagePrefix ("DATA_EC privateKeyBytes : ");
-      gslogMessageHex (pBytes, 0, size, 0, 0, NULL);
+      gslogMessageHex ((char*) pBytes, 0, size, 0, 0, NULL);
```

Comment:
`pBytes` defined as a `unsigned char *`, do we need/have to cast it to just a `char*` ?

Reply:
Hello, yes the compilation fails if I dont cast to `(char*)`. Our other debug code within `#ifdef` value also casts to `(char*)`.

Reply:
ok, I got it.